### PR TITLE
Adding release notes for product recommendations

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds a release notes page to the documentation. We will link to this page from the product recommendations marketplace page.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/release-notes.html

### CODE REVIEW

File: src/recommendations/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+-  {:.new}Added a status column on grid when selecting Recommendation units in Page Builder
+-  {:.fix}Fixed an issue with incorrect http/https protocol in product and image URLs
+
+## **3.0.1** for Magento Commerce (EE) and Commerce on Cloud (ECE) 2.3.1
+
+This is a major version release. You will need to [edit your project's root composer.json file](https://devdocs.magento.com/recommendations/install-configure.html#update-your-product-recommendations-installation)
+
+-  {:.new}Page Builder support. With the Page Builder integration, you can accurately and granularly place Recommendation units in any arbitrary location on Page Builder-authored content, as well as style headings and units themselves. This new feature is documented here: [Page Builder support](https://docs.magento.com/user-guide/marketing/page-builder-add-product-recs.html)
+
+-  {:.new}Ability to fetch Product Recommendations from alternate SaaS environments.
```
Comment (keharper) [inline src/recommendations/release-notes.md:25]: Consider deleting "Ability to" and leave the rest as a sentence fragment, like you did for the first bullet item. Add a brief explanatory sentence, then close with a link to the docs.

File: src/recommendations/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+The release notes include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## **3.0.2** for Magento Commerce (EE) and Commerce on Cloud (ECE) 2.3.1
+
+-  {:.new}Added a status column on the grid when selecting Recommendation units in Page Builder
+-  {:.fix}Fixed an issue with incorrect http/https protocol in product and image URLs
```
Comment (keharper) [inline src/recommendations/release-notes.md:17]:
```suggestion
-  {:.fix}Fixed an issue with incorrect http/https protocols in product and image URLs
```

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Pull request merged**